cmd/org/clusters.go

```diff
@@ -138,6 +138,7 @@ func printClusters(items []*accountsv1.Subscription) {
 		for _, item := range items {
 			subscription := map[string]string{
 				"cluster_id":   item.ClusterID(),
+				"external_id":  item.ExternalClusterID(), //external ID
```

Comment:
The function name is well defined and tells us what it is, therefore we don't need this comment. Same applies to line 156